apps/namadillo/src/atoms/integrations/functions.ts

```diff
@@ -56,7 +59,15 @@ registry.assets.push(
   namadaAssets
 );
 
-const mainnetChains: ChainRegistryEntry[] = [celestia, cosmos, osmosis, stride];
+const mainnetChains: ChainRegistryEntry[] = [
+  celestia,
+  cosmos,
+  osmosis,
+  stride,
+  neutron,
+  noble,
+  nyx,
+];
```

Comment:
The magic is here, lets remember to add the new chains here as well when he have even more tokens. Or think in a smart solution to get this dynamically from somewhere

Reply:
Can we also add a note above this like: "This is the array we must update to add new assets" or something similar